### DISCUSSION TITLE
0135. 分发糖果

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#121] [Comments]
# 0135. 分发糖果

[https://algo.itcharge.cn/solutions/0100-0199/candy/](https://algo.itcharge.cn/solutions/0100-0199/candy/)

**Comment by Nicholasyxx** [comment]
应该是第i 个孩子目前拥有的糖果数量

**Reply by itcharge** [reply]
感谢指正，文中已更新。
